geometry_msgs/Point center
float64 radius
float64 cut_force_k
float64 cut_force_d
geometry_msgs/Vector3 direction
geometry_msgs/Vector3 plane
